AbletonMCP_Remote_Script/__init__.py

```
@@ -1060,3 +1880,64 @@ def get_browser_items_at_path(self, path):
+            if not app:
+                raise RuntimeError("Could not access Live application")
+            if not hasattr(app, "browser") or app.browser is None:
+                raise RuntimeError("Browser is not available in the Live application")
+
+            query = (query or "").strip().lower()
+            if not query:
+                raise ValueError("query is required")
+
+            try:
+                max_results = int(max_results)
+            except:
```

Ruff (0.15.14)

[error] Do not use bare `except`

(E722)